starcheck/pcad_att_check.py

```diff
@@ -0,0 +1,44 @@
+import hopper
+
+def make_pcad_attitude_check_report(backstop_file, or_list_file=None,
+                                    ofls_characteristics_file=None, out=None,
+                                    ):
+    """
+    Make a report for checking PCAD attitudes
+
+    """
+
+    # Run the commands and populate attributes in `sc`, the spacecraft state.
+    # In particular sc.checks is a dict of checks by obsid.
+    # Any state value (e.g. obsid or q_att) has a corresponding plural that
+    # gives the history of updates as a dict with a `value` and `date` key.
+    # This does not define the initial state from continuity at this time.
+    sc = hopper.run_cmds(backstop_file, or_list_file, ofls_characteristics_file,
+                         initial_state=None)
```

Comment:
The `initial_state` does need to include the initial quaternion or else you can have a case where the first maneuver ends after the second one (if it is short) because the first initial attitude is wrong.  I actually ran into this.  Not clear what kind of structures you can pass in via inline Python, but `make_pcad_attitude_check_report` can have `q1` .. `q4` params or whatever.